tests/unittests/Makefile

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright (c) Microsoft. All rights reserved
+#
+
+include $(CLEAR_VARS)
+
+CLANG := clang++
+
+SRCDIR := Foundation
+SRCFILES := $(wildcard $(SRCDIR)/*.m*)
+RF_SRCFILES := $(wildcard $(SRCDIR)/ReferenceFoundation/*.mm)
+
+OUTDIR := ../../build/Tests/UnitTests/Foundation/OSX
+OUTTARGETS := $(addsuffix .o, $(addprefix $(OUTDIR)/, $(basename $(notdir $(SRCFILES)))))
+OUTTARGETS += $(addsuffix .o, $(addprefix $(OUTDIR)/ReferenceFoundation_, $(basename $(notdir $(RF_SRCFILES)))))
```

Comment:
>, $( [](start = 72, length = 4)

nit: Make is unusually sensitive to spaces in `$(function a,b,c)` constructs; it will insert unnecessary spaces in the output unless you remove them around the commas.